packages/website/pages/explore/[id].tsx

```diff
@@ -116,7 +116,7 @@ const SingleView = () => {
           bid:{
               amount: utils.format.parseNearAmount(amount),
               bidder: accountId,
-              recipient: nft.owner?.id,
+              recipient: nft?.id,
```

Comment:
why nft id as recipient?

Reply:
does `nft?.onwer` not work anymore?

Reply:
It is because we are referring `nft_id` as the `recipient` in the contract code. But I also think `recipient` should be the owner, not the `nft_id`. So if we are gonna change this, first we have change to it in the contract side.

https://github.com/curaOS/contracts/blob/1b0f148e38af5c316a4a4d8aaa2a1b2adda06851/as/cNFT/assembly/models/market.ts#L12-L13
https://github.com/curaOS/contracts/blob/1b0f148e38af5c316a4a4d8aaa2a1b2adda06851/as/cNFT/assembly/market.ts#L89

Reply:
but in the contract id should simply be number of minted token